packages/mux-player/src/template.ts

```diff
@@ -0,0 +1,331 @@
+      --media-button-icon-height: auto;
+      pointer-events: none;
+      width: 100%;
+      display: flex;
+      flex-flow: row;
+      align-items: center;
+      justify-content: center;
+    }
+    .mxp-center-controls media-play-button {
+      --media-control-background: transparent;
+      --media-control-hover-background: transparent;
+      padding: 0;
+      width: min(9%, 90px);
+    }
+    .mxp-center-controls media-seek-backward-button,
+    .mxp-center-controls media-seek-forward-button {
+      --media-control-background: transparent;
+      --media-control-hover-background: transparent;
+      padding: 0;
+      margin: 0 10%;
+      width: min(7%, 70px);
+    }
+    media-loading-indicator {
+      --media-loading-icon-width: 100%;
+      --media-button-icon-height: auto;
+      pointer-events: none;
+      position: absolute;
+      width: min(15%, 150px);
+      display: flex;
+      flex-flow: row;
+      align-items: center;
+      justify-content: center;
+    }
+    /* Intentionally don't target the div for transition but the children
+     of the div. Prevents messing with media-chrome's autohide feature. */
+    media-loading-indicator + div * {
+      transition: opacity 0.15s;
+      opacity: 1;
+    }
+    media-loading-indicator[media-loading]:not([media-paused]) ~ div > * {
+      opacity: 0;
+      transition-delay: 400ms;
+    }
+    media-volume-range {
+      width: min(100%, 100px);
+    }
+    media-time-display {
+      white-space: nowrap;
+    }
+  </style>
+  <media-controller style="${getChromeStylesFromProps(props)}">
+    <mux-video
+      slot="media"
+      crossorigin
+      playsinline
+      ${props.autoplay ||
+      props.streamType === StreamTypes.LIVE ||
+      props.streamType === StreamTypes.LL_LIVE
+        ? "autoplay"
+        : ""}
+      ${props.muted ? "muted" : ""}
+      ${props.debug ? "debug" : ""}
+      ${props.preferMse ? "prefer-mse" : ""}
+      ${props.startTime != null ? `start-time="${props.startTime}"` : ""}
+      poster="${props.poster ?? getPosterURLFromPlaybackId(props.playbackId)}"
+      ${props.playbackId ? `playback-id="${props.playbackId}"` : ""}
+      ${props.envKey ? `env-key="${props.envKey}"` : ""}
+      ${props.streamType ? `stream-type="${props.streamType}"` : ""}
+      ${props.metadata?.video_id
+        ? `metadata-video-id="${props.metadata.video_id}"`
+        : ""}
+      ${props.metadata?.video_title
+        ? `metadata-video-title="${props.metadata.video_title}"`
+        : ""}
+      ${props.metadata?.viewer_user_id
+        ? `metadata-viewer-user-id="${props.metadata.viewer_user_id}"`
+        : ""}
+    >
+      <track
+        label="thumbnails"
+        default
+        kind="metadata"
+        src=${getStoryboardURLFromPlaybackId(props.playbackId)}
+      />
+    </mux-video>
+    ${renderable("chromeRenderer", ChromeRenderer, props)}
+  </media-controller>
+`;
+
+export const ChromeRenderer = (props: MuxTemplateProps) => {
```

Comment:
This is a really good case study for media-chrome themes. We should find some time to chat through how we can do that well and align the two so future media chrome themes could be used with mux player. Unless we know of any blockers to doing that.